base/gmp.jl

```diff
@@ -514,8 +514,7 @@ hex(n::BigInt) = base(16, n)
 function base(b::Integer, n::BigInt)
     2 <= b <= 62 || throw(ArgumentError("base must be 2 ≤ base ≤ 62, got $b"))
     p = ccall((:__gmpz_get_str,:libgmp), Ptr{UInt8}, (Ptr{UInt8}, Cint, Ptr{BigInt}), C_NULL, b, &n)
-    len = Int(ccall(:strlen, Csize_t, (Cstring,), p))
-    String(pointer_to_array(p,len,true))
+    unsafe_string_wrapper(p, true)
```

Comment:
No.  Or rather, it doesn't matter.  `Cstring` does not check for embedded nuls if it is passed a pointer argument — there is no way to do so, unless you have independent information on the length.   When you pass a pointer, `Cstring` is equivalent to `Ptr{UInt8}`.